networking/hardware_networks/using-dpdk-and-rdma.adoc

```diff
@@ -28,6 +39,13 @@ include::modules/nw-sriov-rdma-example-mellanox.adoc[tag=content]
 [id="additional-resources_using-dpdk-and-rdma"]
 == Additional resources
 
+* xref:../../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-about-the-profile-creator-tool_cnf-create-performance-profiles[Creating a performance profile].
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#adjusting-nic-queues-with-the-performance-profile_cnf-master[Reducing NIC queues using the Node Tuning Operator].
```

Comment:
After a rebase, this xref is not found. This is because the `cnf-performance-addon-operator-for-low-latency-nodes` section was renamed in the topic map [yesterday in PR 47965](https://github.com/openshift/openshift-docs/pull/47965/files#diff-7c09a5ccc63f204d75ed89dd359a340a43551a4a790ae36fb9c64f6c5149211bL2240-R2241).

That PR was merged for main, but at the time of writing this comment, the 4.11 automatic cherry pick PR https://github.com/openshift/openshift-docs/pull/48034 is not merged.

Please rebase the PR and update this xref to point to the new target section location. I'm not sure what section to reference so I can't provide an xref example. This is because I can't see a section called "Reducing NIC queues using the Node Tuning Operator" in the renamed "Low latency tuning" page in my preview based on main. There is a "Reducing NIC queues using the Performance Addon Operator", but I don't want to assume that it is the intended reference.

Note that the updated xref will probably cause a build check error in the cherry pick of this PR (46316) until https://github.com/openshift/openshift-docs/pull/48034 is merged.

CC: @johnwilkins and @stevsmit